MrtCore/winui_desktop_packaged_app/ClassLibrary/Class.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text;
+using Microsoft.ApplicationModel.Resources;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.

```

Comment:
does this have anything to do with WinUI?

Reply:
removed